### PR TITLE
BUGFIX use tax data from the quote object

### PR DESCRIPTION
**Pull request by andrewandante**
Apply fix for 1.2.0

### CODE REVIEW

**Comment by andrewandante** [comment]
Updated to 1.2.1 so that we can pass the marketplace standards that now forbid include/require

File: view/adminhtml/templates/order/view/lwdata.phtml [hunk 1]
```diff
@@ -1,29 +1,29 @@
 <?php /** @var $block \LendingWorks\RetailFinance\Block\Adminhtml\Order\View\LendingWorksData */ ?>
 <section class="admin__page-section">
     <div class="admin__page-section-title">
-        <span class="title"><?php echo $block->escapeHtml(__('Lending Works Data')) ?></span>
+        <span class="title"><?= $block->escapeHtml(__('Lending Works Data')) ?></span>
```
Comment (chinthakagodawita) [inline view/adminhtml/templates/order/view/lwdata.phtml]: Why this change? This requires short tags to be enabled on the system, which they might not be

Comment (andrewandante) [reply view/adminhtml/templates/order/view/lwdata.phtml]: Magento linter recommended it ¯\_(ツ)_/¯ can remove
